main.go

```diff
@@ -0,0 +1,111 @@
+package main
```

Comment:
Please delete this file... seems leftover from early experiments?

Reply:
Yikes! `git add` betrayed me